browser/brave_wallet/asset_ratio_service_factory.h

```diff
@@ -24,6 +24,9 @@ class AssetRatioServiceFactory : public BrowserContextKeyedServiceFactory {
   static AssetRatioService* GetServiceForContext(
       content::BrowserContext* context);
   static AssetRatioServiceFactory* GetInstance();
+  static void BindForContext(
+      content::BrowserContext* context,
+      mojo::PendingReceiver<mojom::AssetRatioService> receiver);
```

Comment:
Include `mojo/public/cpp/bindings/pending_receiver.h` for this and same for other *factory.h.